src/app/about/page.tsx

```diff
@@ -25,6 +26,22 @@ export default async function About() {
 
   return (
     <Container maxWidth="lg" className="about-page">
+      <Head>
+        {teamMemberData.map((teamMember: TeamMember, index: number) => (
```

Comment:
I wonder if it would be cleaner to move this into its own function outside of the returned JSX. Perhaps something like:

```javascript
export default async function About() {
  const teamMemberData = await getTeamMemberData();

  const getJsonLd = () => {
    return teamMemberData.map((teamMember: TeamMember, index: number) => (
      <script type="application/ld+json" key={index}>
        {`{
          "@context": "https://schema.org",
          "@type": "Person",
          "name": "${teamMember.name}",
          "jobTitle": "${teamMember.job_title}",
          "description": "${teamMember.bio || ''}",
          "url": "https://www.codeforbc.ca/about",
          "image": "https://codeforbc-web.vercel.app${teamMember.profile_image_link || ''}",
          "sameAs": ["${teamMember.linkedin_link}", "${teamMember.github_link || ''}"]
        }`}
      </script>
    ));
  };

  return (
    <Container maxWidth="lg" className="about-page">
      <Head>{getJsonLd()}</Head>
      {/* ... */}
    </Container>
  );
}

```

Reply:
addressed in [13de453](https://github.com/CodeForBc/codeforbc-web/pull/23/commits/13de45333e59aec4ead6adeed4e3fcd6f8e69aa7)